packages/analytics/client/src/schema/validation/schema_to_io_ts.ts

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import * as t from 'io-ts';
+import type { RootSchema, SchemaArray, SchemaObject, SchemaValue } from '../types';
+import { excess } from './excess';
```

Comment:
The main reason I called it `excess` is because that was the name folks in the `io-ts` library referred to it in the original issue where I found the code I based my solution upon: https://github.com/gcanti/io-ts/issues/322

Happy to rename it if you think it's worth it. What would that name be?